src/sortperm.jl

```diff
@@ -23,7 +23,7 @@ function best_perm_estimate(perms, cols)
     return (bestmatch, bestperm)
 end
 
-function sortpermby(t, by; cache=false)
+function sortpermby(t, by; cache=false, return_rows=false)
```

Comment:
Name suggestion: `return_sorted`?

Reply:
That may be confusing though because the rows are not actually sorted (they are sorted according to `perm`), maybe `return_keys`? We end up calling that thing `keys` very often.